specification/maps/data-plane/Search/preview/2022-09-01-preview/examples/ReverseGeocoding.json

```diff
@@ -0,0 +1,86 @@
+{
+  "parameters": {
+    "api-version": "2022-09-01-preview",
+    "query": "-122.138681,47.630358",
+    "AzureKey": "[subscription-key]"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "geoJsonData": {
+          "type": "FeatureCollection",
+          "features": [
+            {
+              "type": "Feature",
+              "properties": {
+                "address": {
+                  "countryRegion": {
+                    "name": "United States"
+                  },
+                  "adminDistricts": [
+                    {
+                      "shortName": "WA"
+                    },
+                    {
+                      "shortName": "King County"
+                    }
+                  ],
+                  "formattedAddress": "15127 NE 24th St, Redmond, WA 98052",
+                  "locality": "Redmond",
+                  "postalCode": "98052",
+                  "addressLine": "15127 NE 24th St"
+                },
+                "type": "Address",
+                "confidence": "Medium",
+                "matchCodes": [
+                  "Good"
+                ],
+                "geocodePoints": [
```

Comment:
For Reverse Geocode, do we still want to show Geocode Points?

Reply:
We have that in the current API due to the legacy reason. We do have the opportunity to fix that, but we could keep it for maximal backward compatibility as well.

Reply:
It's a good question. I might need to get more knowledge whether users are using this field.
Currently, both forward and RevGC have this field.
If platform service would like to remove this field, will it be removed in forward GC API as well?
We would like to keep the response format of forward and Rev GC consistent.
For the forward GC API in Azure map, we are proxying platform service, so it's important to know whether any change might happen in the upstream data.

Reply:
According to our discussion, we will still show Geocode Points in RevGC api.